pkgs/by-name/go/godini/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  godini,
+  lib,
+  nix-update-script,
+  testers,
+}:
+
+buildGoModule rec {
+  pname = "godini";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "bilbilak";
+    repo = "${pname}";
```

Comment:
`repo = pname;` is an antipattern as the two are arbitrary and not necessarily equal